.github/workflows/staging_linter.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       - name: Run main database migrations
         run: mix ecto.migrate --log-migrator-sql
       - name: Run database tenant migrations
-        run: mix ecto.migrate --migrations-path priv/repo/postgres/migrations
+        run: mix ecto.migrate --migrations-path lib/realtime/tenants/repo/migrations
```

Comment:
these migrations are only part of RLS, so they should be inside the `lib/extensions/postgres_cdc_rls` directory

Reply:
done (e.g. `lib/extensions/postgres_cdc_rls/repo/migrations/20211116024918_create_realtime_subscription_table.ex`)